CONTRIBUTING.md

```diff
@@ -283,6 +283,26 @@ writeup][writeup]
 For installation and configuration in vanilla emacs, see
 https://scalameta.org/metals/docs/editors/emacs.html
 
+## Documentation
+
+We maintain two main kinds of documentation: tutorials and manuals meant as user
+documentation and API documentation meant for developers.
```

Comment:
Should we also mention RFCs/ADRs (which seem to fall somewhere in between)?

Reply:
Ah yeah. Good idea.

Reply:
how do you find https://github.com/informalsystems/apalache/pull/2026/commits/1a4b77c1623bb4cf480766d4f477d0cf5619db1e ?

Reply:
Excellent, just a word fix in the comment below.